python/pyspark/ml/classification.py

```diff
@@ -490,7 +411,7 @@ class RandomForestClassificationModel(TreeEnsembleModels):
 
 @inherit_doc
 class GBTClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, HasMaxIter,
-                    DecisionTreeParams, HasCheckpointInterval):
+                    GBTParams, HasCheckpointInterval, HasStepSize):
```

Comment:
Should be also extend ```TreeClassifierParams```? I know currently only ```impurity``` in ```TreeClassifierParams``` and ```GBTClassifier.setImpurity``` should NOT be used, but I think we should keep the same inheritance relationships between Scala & Python. May be one day we will have more params in ```TreeClassifierParams`` and ```GBTClassifier``` will use those params.